www/source/tutorials/getting-started/windows/create-plan.html.md.erb

```diff
@@ -0,0 +1,61 @@
+---
+title: Creating your first plan
+---
+
+<ul class="main-content--version">
+  <li>Current version:</li>
+  <li><a href="/tutorials/getting-started/mac/create-plan">Mac</a></li>
+  <li><a href="/tutorials/getting-started/linux/create-plan">Linux</a></li>
+  <li><a class="active" href="/tutorials/getting-started/windows/create-plan">Windows</a></li>
+</ul>
+<hr />
+
+# Create your first plan
+<%= partial "/shared/create_plan_common", locals: { is_windows: true } %>
+
+## Entering the studio and doing an initial build
+The studio is a minimal, clean, [chrooted](https://en.wikipedia.org/wiki/Chroot) shell environment for you
+to build and test your Habitat packages. The studio itself is a Habitat package, and all of the dependent packages for the studio will be downloaded,
+unpacked, and installed. The studio will also import the secret origin key created during `hab setup`. When running Mac OS X on your host machine, the studio runs
+inside a Docker container as root.
+
+Packages are built in the studio through the `hab-plan-build` script, which handles creating Habitat packages from plan files.
+The `hab-plan-build` script looks for `plan.sh` either in the current directory, or in a `./habitat` directory.
+
+> Note: The current directory you are in on your host machine will become the `/src` directory when you enter the studio.
+In this tutorial, because we are copying source files from within our repo into our package, we must enter the studio at the
+`/mytutorialapp` directory level so that our `/source` directory is included in the chrooted studio environment.
+
+1. Change directories to the root directory for your project.
+
+       cd ~/habitat-example-plans/mytutorialapp/
+
+2. Make sure Docker for Mac is running and then enter into the studio environment.
```

Comment:
got it in https://github.com/habitat-sh/habitat/pull/1350/commits/183b20d768742c1abd72192dc9cc26a1f2faa2ef